### PR TITLE
added model for /operators call

### PR DESCRIPTION
**Pull request by jeroen7s**
Jira: https://vikingco.atlassian.net/browse/UWA-415

### CODE REVIEW

File: paths/index.yaml [hunk 1]
```diff
@@ -79,3 +79,5 @@
   $ref: ./events/list.yaml
 /events/{id}:
   $ref: ./events/details.yaml
+/operators:
```
Comment (bmoelans) [inline paths/index.yaml]: I would think /sims/operators is better, what do you think @JanJoris

File: definitions/index.yaml [hunk 2]
```diff
@@ -106,3 +106,5 @@ CreatePosCustomer:
   $ref: ./point_of_sale/create_customer.yaml
 Event:
   $ref: ./events/event.yaml
+Operator:
+  $ref: ./telco/operator.yaml
```
Comment (bmoelans) [inline definitions/index.yaml]: newline

File: definitions/telco/operator.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+type: object
+title: Operator
+properties:
+  id:
```
Comment (bmoelans) [inline definitions/telco/operator.yaml]: do we need to expose id?

File: definitions/telco/operator.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+    type: string
+  name:
+    type: string
+  physical:
```
Comment (bmoelans) [inline definitions/telco/operator.yaml]: do we need to expose this @Kjoep @JanJoris ? If so, it should be enum.

Comment (jeroen7s) [reply definitions/telco/operator.yaml]: if we make it an enum, wouldn't it be useless to have it configurable in the database?

Comment (jeroen7s) [reply definitions/telco/operator.yaml]: I'm not sure if it's needed, same for id, depends on what you'd like to show in the frontend and what you'd like to pass as donor operator to create_subscription.

File: paths/index.yaml (hunk 1 above)
Comment (jeroen7s) [reply paths/index.yaml]: everything under sims seem to deal with specific sim-cards, and so also require a login of a user which owns that sim, this call needs to be open

Comment (Kjoep) [reply paths/index.yaml]: Yeah I don't see the relation with /sims either.  /operators seems fine to me.

File: definitions/telco/operator.yaml (hunk 4 above)
Comment (Kjoep) [reply definitions/telco/operator.yaml]: I suppose we need to pass the ID as the donor ID when we ask for mnp? 'name' _could_ be localized, that would not be a good key.

File: definitions/telco/operator.yaml (hunk 3 above)
Comment (Kjoep) [reply definitions/telco/operator.yaml]: I suppose we need to pass the ID as the donor ID when we ask for mnp? 'name' could be localized, that would not be a good key

File: definitions/telco/operator.yaml (hunk 4 above)
Comment (Kjoep) [reply definitions/telco/operator.yaml]: Not sure.  We probably don't need it on the front end.

File: definitions/telco/operator.yaml (hunk 3 above)
Comment (bmoelans) [reply definitions/telco/operator.yaml]: we need to pass to Comarch the physical operator, so in UWA we map Operator to physical. So for logging and readability, I would suggest to pass just name, just as we do with other properties that are an enum.

File: definitions/telco/operator.yaml (hunk 4 above)
Comment (bmoelans) [reply definitions/telco/operator.yaml]: if not needed in front-end, leave it out, we can add it later, but physical is just an internal matter of UWA to know what to pass

File: paths/index.yaml (hunk 1 above)
Comment (JanJoris) [reply paths/index.yaml]: `/operators` is fine, it's that or `telco/operators` but let's not push it.

File: definitions/telco/operator.yaml (hunk 3 above)
Comment (JanJoris) [reply definitions/telco/operator.yaml]: Agreed

File: definitions/telco/operator.yaml (hunk 4 above)
Comment (JanJoris) [reply definitions/telco/operator.yaml]: idd, leave it out.

File: definitions/telco/operator.yaml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+type: object
+title: Operator
+properties:
+  name:
+    type: string
```
Comment (bmoelans) [inline definitions/telco/operator.yaml]: newline

**Pull request merged**